reflekt-plugin/src/main/kotlin/org/jetbrains/reflekt/plugin/generation/code/generator/ReflektImplGenerator.kt

```diff
@@ -52,7 +54,56 @@ class ReflektImplGenerator(private val libraryQueriesResults: LibraryQueriesResu
                     body = statement("return %T()", generator.typeName),
                 )
             })
+
             addNestedTypes(innerGenerators.map { it.generate() })
+
+            builder.addProperty(
+                "reflektClasses",
+                MAP.parameterizedBy(ClassName("kotlin.reflect", "KClass").parameterizedBy(STAR), ReflektClass::class.asTypeName().parameterizedBy(STAR)),
+            )
+
+            builder.addInitializerBlock(buildCodeBlock {
+                addStatement("val m = HashMap<KClass<*>, ReflektClassImpl<*>>()")
```

Comment:
Should we link the `m` variable between statements? I mean if we change it here, we also should change it know 72, 105 code rows and it could be not clear

Reply:
The same thing is for `ReflektClassImpl`. Also probably the great idea is to leave a comment about the initial `ReflektClassImpl` class to indicate which fields should be implemented

Reply:
> Should we link the `m` variable between statements? I mean if we change it here, we also should change it know 72, 105 code rows and it could be not clear

Such errors will be easily handled by tests. It's OK for code generation consist from undocumented string literals because compiler will always report all errors made.

Reply:
> The same thing is for `ReflektClassImpl`. Also probably the great idea is to leave a comment about the initial `ReflektClassImpl` class to indicate which fields should be implemented

Agree.